apis/outlook/mail/manage/tool.gpt

```diff
@@ -9,4 +9,4 @@ Type: context
 
 You have access to a collection of operations in the Microsoft Outlook Mail API.
 Do not output mail folder IDs or message IDs because they are not helpful for the user. The message IDs are needed for getting message details, deleting a message, or moving a message.
-When printing a list of messages for the user, include the body preview. When printing a single message and its details, print the full body.
+When printing a list of messages for the user, include the body preview. When printing a single message and its details, print the full body. Always include the email link.
```

Comment:
Not sure if telling it to print link with markdown format will yield better visual result

Reply:
I think we should do this just to be explicit. LLMs are pretty good at writing markdown.